third_party/android_deps/libs/com_brave_playlist/README.chromium

```diff
@@ -0,0 +1,4 @@
+Name: Brave Playlist
+URL: https://github.com/brave-experiments/playlist-kotlin
```

Comment:
Brave owned code should not be added as a third_party lib. The code should either live directly in brave-core (preferred) or if it's used outside of brave-core then it should be added through DEPS and in both cases should be built by gn